src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> tasks;
+    public TaskList() {
```

Comment:
Missing javadocs and would personally have an empty line in between tasks and tasklist()